mappings/net/minecraft/server/world/ChunkTicketManager.mapping

```diff
@@ -37,7 +37,7 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 	METHOD method_14051 handleChunkLeave (Lnet/minecraft/class_4076;Lnet/minecraft/class_3222;)V
 		ARG 1 pos
 		ARG 2 player
-	METHOD method_14052 getLevelCount ()I
+	METHOD method_14052 getSpawningChunkCount ()I
```

Comment:
This isn't used anywhere else? I thought this just returned the number of chunk levels there are, which aren't just used for spawning (there's also levels for ticking, worldgen, structures, etc.).

Reply:
Yes, that is this method's exclusive usage.